snake/app.py

```diff
@@ -53,10 +53,11 @@ def run(self):
                 else:
                     filtered_events.append(event)
             self.currentScene.ProcessInput(filtered_events, pressed_keys)
-            if time >= self.fps:
+            if time >= self.fps and self.isPaused != True:
```

Comment:
Do we need this condition here, because you already cehck on line 62 and time will not be increment anymore

Reply:
That's true ! I will delete this condition